### ISSUE TITLE
Issue ID: DAC_Relationship_Not_Clear2

### ISSUE THREAD

**Issue by VDuczekW** [guardian/dotcom-rendering#5065]
Report Link: https://drive.google.com/file/d/1lm8L-zQ3Te5nCnJ0EaZMZ9CyikYM9rdT/view?usp=sharing

Full issue list: https://docs.google.com/spreadsheets/d/1vWQtZfD49uq7uNCNELuYN_HcIj-BKiE4t4Hwda3tJaQ/edit#gid=0

**Comment by bryophyta** [comment]
Quiz questions are now marked up as `<li>` elements ([PR in atoms-rendering](https://github.com/guardian/atoms-rendering/pull/498)). This introduced a bug which should be fixed separately.
